Sources/performance-worker/WorkerService.swift

```diff
@@ -41,12 +55,42 @@ final class WorkerService: Grpc_Testing_WorkerService.ServiceProtocol, Sendable
       self.role = role
     }
 
-    init(server: GRPCServer) {
-      self.role = .server(server)
+    func getServer() -> GRPCServer? {
+      switch self.role {
+      case let .server(serverState):
+        return serverState.server
+      default:
+        return nil
+      }
     }
 
-    init(client: GRPCClient) {
-      self.role = .client(client)
+    mutating func initialServerStats(set newValue: ServerStats? = nil) -> ServerStats? {
```

Comment:
These aren't always the initial (i.e. when the server started) stats. 

I think `serverStats(replaceWith newStats: ServerStats? = nil) -> ServerStats?` is a better spelling here.